ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -130,32 +147,6 @@ export default class UnlockPage extends Component {
     }
   }
 
-  renderSubmitButton() {
-    const style = {
-      backgroundColor: 'var(--color-primary-default)',
-      color: 'var(--color-primary-inverse)',
-      marginTop: '20px',
-      height: '56px',
-      fontWeight: '500',
-      boxShadow: 'none',
-      borderRadius: '100px',
-    };
-
-    return (
-      <Button
-        type="submit"
-        data-testid="unlock-submit"
-        style={style}
-        disabled={!this.state.password}
-        variant="contained"
-        size="large"
-        onClick={this.handleSubmit}
-      >
-        {this.context.t('unlock')}
-      </Button>
-    );
-  }
```

Comment:
Replacing separate function in favor in inline JSX in the render function. We are also removing the inline style and updating the component to use a design system component